docs/src/model_creation/network_analysis.md

```diff
@@ -31,10 +31,10 @@ In the [Introduction to Catalyst](@ref introduction_to_catalyst)
 tutorial we showed how the above network could be visualized as a
 species-reaction graph. There, species are represented by the nodes of the graph
 and edges show the reactions in which a given species is a substrate or product.
-```julia
-g = Graph(repressilator)
+```@example s1
+using Catalyst, GraphMakie, GLMakie
+g = plot_network(repressilator)
```

Comment:
The generated graph here and in the tutorial seems to have lines crossing unnecessarily?